arviz/plots/backends/bokeh/pairplot.py

```diff
@@ -66,7 +70,7 @@ def plot_pair(
 
         if ax is None:
             ax = bkp.figure(
-                width=int(figsize[0] * dpi), height=int(figsize[1] * dpi), **backend_kwargs
+                width=int(figsize[0] * dpi), height=int(figsize[1] * dpi), **backend_kwargs, tooltips=TOOLTIPS
```

Comment:
This does the job, but maybe we could add tooltip into backend_kwargs with `.setdefault`, if user wants to change the default behaviour.